.teamcity/src/main/kotlin/promotion/PromotionProject.kt

```diff
@@ -42,6 +43,8 @@ class PromotionProject(branch: VersionedSettingsBranch) : Project({
         password("env.DOTCOM_DEV_DOCS_AWS_SECRET_KEY", "%dotcomDevDocsAwsSecretKey%")
         password("env.ORG_GRADLE_PROJECT_sdkmanToken", "%sdkmanToken%")
         param("env.JAVA_HOME", javaHome(BuildToolBuildJvm, Os.LINUX))
+        // https://github.com/gradle/gradle-private/issues/4504
+        param("env.JDK8", javaHome(OpenJdk8, Os.LINUX))
```

Comment:
❓ Looking at the PRs 
* #30726
* #30752

I'm wondering if we should also include `JDK11` right away.

And in the latter PR we should add `JDK17` here in the same way